instance/models/instance.py

```diff
@@ -157,6 +157,11 @@ class Instance(ValidateModelMixin, models.Model):
         blank=True,
         help_text='Custom tags associated with the instance.',
     )
+    accepted_privacy_policy = models.DateTimeField(
```

Comment:
@jamestait, this field should be read-only in the admin interface.

Reply:
Given we're intending to ask customers to agree to the Privacy Policy by email, do we want to keep this editable, or perhaps editable if empty, so we can add this date manually when we get customers' acceptance?

Reply:
@jamestait, we could do that via the CLI and keep the admin interface read-only. But that would mean Gabriel won't be able to update this if he wants to.

Reply:
I've just pushed that change. :slightly_smiling_face:

Reply:
> do we want to keep this editable, or perhaps editable if empty, so we can add this date manually when we get customers' acceptance

This looks like the best way to do this.